Shared/UI/Views/DisplayableImageView.swift

```diff
@@ -185,7 +185,12 @@ class DisplayableImageView: BaseView {
     @MainActor
     func set(image: UIImage?, state: State) async {
         self.state = state
-        self.imageView.image = await image?.byPreparingForDisplay()
+
+        if let preparedImage = await image?.byPreparingForDisplay() {
+            self.imageView.image = preparedImage
+        } else {
+            self.imageView.image = image
+        }
```

Comment:
HEIC images are viewed as "corrupt" by this function. It prints out a harmless warning and fails. If the prepare fails we just use the original image.